.github/workflows/ci.yml

```diff
@@ -38,6 +38,14 @@ jobs:
           chmod +x bin/tinysearch
           echo "$(pwd)/bin" >> ${GITHUB_PATH}
 
+      - name: ⏬ Install cavif
+        run: |
+          set -e
+          mkdir cavif
+          curl -sSL https://github.com/kornelski/cavif-rs/releases/download/v1.3.3/cavif-1.3.3.zip | tar xz --directory=cavif
```

Comment:
This is a zip. You need `unzip`.

Reply:
Yeah, I realized that, too. It was late when I did that. 😆